subprojects/core/src/main/java/org/gradle/api/internal/tasks/execution/ExecuteActionsTaskExecuter.java

```diff
@@ -246,7 +246,7 @@ public void visitLocalState(LocalStateVisitor visitor) {
         }
 
         @Override
-        public boolean includeAddedOutputs() {
+        public boolean isAllowOverlappingOutputs() {
             return false;
```

Comment:
That seems to be the wrong way around: tasks allow overlapping outputs, artifact transforms don't.

Reply:
Good catch.